app/controllers/Application.scala

```diff
@@ -91,4 +91,27 @@ class Application(implicit val wsClient: WSClient, val db: MetricsDB) extends Co
   }
 
   def getNewspaperBookList = APIAuthAction(APIResponse(db.getDistinctNewspaperBooks))
+
+  def getArticlesWithWordCounts() = APIAuthAction { req =>
+    APIResponse {
+      for {
+        articlesWithoutCommissionedLength <- db.getArticlesWithWordCounts(false)(MetricsFilters(req.queryString))
+        articlesWithCommissionedLength <- db.getArticlesWithWordCounts(true)(MetricsFilters(req.queryString))
```

Comment:
Perhaps
`db.getArticlesWithWordCounts(withCommissionedLength = false)`
`db.getArticlesWithWordCounts(withCommissionedLength = true)`?
As it explicitly tells the reader what the hardcoded boolean represents?